community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -529,8 +529,24 @@ public String providerName()
         }
     }
 
-    @Description( "Index provider to use when creating new indexes." )
-    public static final Setting<String> default_schema_provider =
+    @Description( "Index provider to use for newly created schema indexes." +
+            "An index provider may store different value types in separate physical indexes." +
+            "lucene-1.0: Store spatial and temporal value types in native indexes, remaining value types in a Lucene index." +
+            "lucene+native-1.0: Store numbers in a native index and remaining value types like lucene-1.0." +
```

Comment:
Spatial and temporal types are still in native indexes with `lucene+native-1.0`, aren't they?

Reply:
Perhaps it's hard to get the structure of this list, because the idea was that each provider refer back to the previous, saying that the remaining values i.e. all except numbers are handled like how they are handled in lucene-1.0 provider, which has spatial/temporal stuff, not lucene directly. Do you think it could be made better?

Reply:
Right, so that's what you mean when you say the rest of the values are stored "like lucene-1.0". I suppose it's good enough for me as it is.